caravel/assets/javascripts/SqlLab/reducers.js

```diff
@@ -1,6 +1,7 @@
 import shortid from 'shortid';
 import * as actions from './actions';
 import { now } from '../modules/dates';
+import * as utils from '../../utils/reducerUtils';
```

Comment:
would be nice to use destructing here: so you don't have to use `utils.addToArr` elsewhere.
`const { addToArr, removeFromArr, alterInArr, alterInObject } from '../../utils/reducerUtils'; `